README.md

```diff
@@ -286,7 +304,7 @@ end
 And setup GraphQL to use the built-in `lazy_resolve` method:
 
 ```ruby
-Schema = GraphQL::Schema.define do
+class GraphqlSchema < GraphQL::Schema
```

Comment:
`GraphqlSchema` could we call it the same way as in the first example? so people understand what has to be changed in the first example to get to the second one